internal/commands/result.go

```diff
@@ -1088,7 +1088,7 @@ func filterResultsByType(results *wrappers.ScanResultsCollection, excludedTypes
 
 func filterScsResultsByAgent(results *wrappers.ScanResultsCollection, agent string) *wrappers.ScanResultsCollection {
 	unsupportedTypesByAgent := map[string][]string{
-		commonParams.DefaultAgent:      {},
+
```

Comment:
change/add test with an agent like Jenkins and that we got results(unit) for scs and container

Reply:
json results